examples/master.js

```diff
@@ -199,13 +201,49 @@ async function startMaster(localView, remoteView, formValues, onStatsReport, onR
 
             // Poll for connection stats
             if (!master.peerConnectionStatsInterval) {
-                master.peerConnectionStatsInterval = setInterval(() => peerConnection.getStats().then(onStatsReport), 1000);
+                master.peerConnectionStatsInterval = setInterval(() => peerConnection.getStats().then(onStatsReport), 10000);
```

Comment:
Why are we increasing this?

Reply:
Previously it was a no-op firing every 1 second. Now, it's actually implemented, firing every 10 seconds.
Prints a lot of stuff.